### PR TITLE
[#56][#58] [UI&Integration] As a user, I can see the thank you screen

### PR DESCRIPTION
**Pull request by phongvhd93**
- Close 
- #56 
- #58 

## What happened 👀

- Add `SubmissionSuccessView`, `LottieVIew` and `confetti` resource
- Add library `lottie-ios` to pod file
- Add UI test

## Insight 📝

When the use submits their answers successfully, the application will show the `Thank you` screen
## Proof Of Work 📹

https://github.com/nimblehq/avishek-phong-kmm-ic/assets/22606906/4beb82c3-52cd-40df-95bf-5aeccb4248b3

### CODE REVIEW

**Review by markgravity — APPROVED** [review]

File: shared/src/commonTest/kotlin/co/nimblehq/avishek/phong/kmmic/presentation.module/LogInViewModelTest.kt [hunk 1]
```diff
@@ -29,8 +29,8 @@ class LogInViewModelTest {
 
     @BeforeTest
     fun setUp() {
-        viewModel = LogInViewModel(logInUseCase)
         Dispatchers.setMain(mainThreadSurrogate)
```
Comment (AVI5HEK) [inline shared/src/commonTest/kotlin/co/nimblehq/avishek/phong/kmmic/presentation.module/LogInViewModelTest.kt]: Could you please check if we still need to call `Dispatchers.setMain(mainThreadSurrogate)`? If not, we should remove it.

File: shared/src/commonTest/kotlin/co/nimblehq/avishek/phong/kmmic/presentation.module/SurveyQuestionViewModelTest.kt [hunk 2]
```diff
@@ -17,32 +27,81 @@ import kotlin.test.Test
...
-    private val mainThreadSurrogate = newSingleThreadContext("UI thread")
+    @Mock
+    private val useCase = mock(classOf<SubmitSurveyAnswerUseCase>())
+
     private lateinit var viewModel: SurveyQuestionViewModel
 
     @BeforeTest
     fun setUp() {
-        Dispatchers.setMain(mainThreadSurrogate)
+        Dispatchers.setMain(TestDispatchersProvider.io)
```
Comment (AVI5HEK) [inline shared/src/commonTest/kotlin/co/nimblehq/avishek/phong/kmmic/presentation.module/SurveyQuestionViewModelTest.kt]: Same here, I think we don't need it anymore.
```suggestion
```

**Review by AVI5HEK — COMMENTED** [review]
The rest LGTM.

**Review by AVI5HEK — COMMENTED** [review]
The rest LGTM.

File: shared/src/commonTest/kotlin/co/nimblehq/avishek/phong/kmmic/presentation.module/SurveyQuestionViewModelTest.kt (hunk 2 above)
Comment (phongvhd93) [reply shared/src/commonTest/kotlin/co/nimblehq/avishek/phong/kmmic/presentation.module/SurveyQuestionViewModelTest.kt]: This was solved by rebasing

File: shared/src/commonTest/kotlin/co/nimblehq/avishek/phong/kmmic/presentation.module/LogInViewModelTest.kt (hunk 1 above)
Comment (phongvhd93) [reply shared/src/commonTest/kotlin/co/nimblehq/avishek/phong/kmmic/presentation.module/LogInViewModelTest.kt]: This was solved by rebasing

**Review by AVI5HEK — APPROVED** [review]

**Review by suho — APPROVED** [review]

**Pull request merged**